testing/matrix-sdk-test-macros/src/lib.rs

```diff
@@ -7,6 +7,11 @@ use syn::parse_macro_input;
 #[proc_macro_attribute]
 pub fn async_test(_attr: TokenStream, item: TokenStream) -> TokenStream {
     let fun = parse_macro_input!(item as syn::ItemFn);
+
+    if !fun.sig.ident.to_string().starts_with("test_") {
```

Comment:
I will never get used to this twitchy convulsive way of reading abbreviations. Presumably it's `function.signature.identifier(?)`? who knows..

Reply:
Yep, that's it! These are unfortunately quite common in the proc-macro's `syn` library…